.github/workflows/test.yml

```diff
@@ -124,6 +51,15 @@ jobs:
       CC: ${{ matrix.build.cc }}
     steps:
       - uses: actions/checkout@v4
+
+      - if: ${{ matrix.build.flavor == '32' }}
+        run: |
+          sudo dpkg --add-architecture i386
+          sudo apt-get update -y
+          sudo apt-get install -y gcc-13:i386
```

Comment:
There's nothing specific about the 32-bit build that requires gcc-13. Vim's CI was already forcing a specific version, so I just updated that part to install the 32-bit variant.